Topics/Tech_Stacks/Postman/Postman_mock_server.md

```diff
@@ -0,0 +1,98 @@
+4. If you choose to make your mock server private, you need to add a [Postman API key](https://learning.postman.com/docs/developer/postman-api/intro-api/) in the request header when sending requests to the mock server: x-api-key:<Your-Postman-API-key>. If you share the collection, others can use their Postman API keys to make calls to the mock server.
+
+After you finish selecting configuration options, select Create Mock Server. Postman displays the details you need to use the mock server. (You can get these details at any time by selecting Mock Servers in the sidebar and selecting the mock server.)
+![](./assets/mock-server-get-url.jpg)
+
+You can change the configuration for a mock server at any time. Select Mock Servers in the sidebar, select a mock server, and select Settings icon Edit Configuration.
+
+Select Copy Mock URL to begin making requests to your mock server.
+
+## Making requests to mock servers
+
+### Send request
+
+1. Use the mock server's URL to make calls to the mock server. Select Mock Servers in the sidebar, select a mock server, and select Copy Mock URL.
+2. Make sure the request you want to mock has at least one saved example. You can send a request and save the response as an example. You can also define a custom example. 
+3. Open a new request tab (or edit the address in an existing tab) and add the mock URL:
+
+```
+https://<mock-id>.mock.pstmn.io/<request-path>
+
+```
+For example: 
+
+```
+https://4bb57fc2-219e-421e-86b4-4ffda6bf1b3b.mock.pstmn.io/matches
+
+```
+Here the mock ULR is the mock ULR you copied from the Mock server details. You can add the request path you want to mock.
+
+When you send a request to the mock server, the mock server sends back a response based on an example with the same path and method. You can provide multiple examples, and Postman will return the one that best matches your request.
+
+### Adding examples to a request
+
+#### Saving a response as an example
+When saving a response in Postman, you have the option to save it as an example.
+
+1. Select Collections in the sidebar.
+
+2. Open a request and select Send.
+
+3. In the response pane, select Save icon Save as Example.
+![](./assets/examples-save-response.jpg)
+
+#### Adding a custom example
+With a custom example, you can define how both the request and the response look, including the status code and response body.
+
+1. Select Collections in the sidebar.
+2. Select the more actions icon More actions icon next to a request and then select Add example.
+3. Enter a name for the example.
+4. Edit the request part of the example.
+- Add any parameters or headers as needed.
+- Enter the request body and select a content type.
+5. Edit the response part of the example.
+- Enter a Status Code (such as 200 or 404).
+- Enter the response body and select a content type.
+- Add any headers as needed.
+6. Select Save to save the example.
+![](./assets/examples-add-custom.jpg)
```

Comment:
It will better if you add a conclusion and additional resources.

Reply:
Oh I forgot to include these. Thank you so much for your thoughtful suggestions. I will add these as suggested.